schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		        }
+	         }
+	     },
+
+       "ORIENTATION" : {
+           "description": "Orientation of the sensor provided in quaternions",
+           "type": "array",
+           "items": {
+               "type": "array",
+               "items" : {
+                  "type": "number",
+                  "minItems": 4 ,
+                  "maxItems": 4
+           }
+          }
+      },
+
+       "DETECTOR_LINE_RATE": {
+           "description": "The line rates of the detector map with the associated start line and time",
+           "type": "array",
+              "items": {
+	               "type": "object",
+	                "properties": {
+	                   "START_LINE": {
+		                     "type": "number"
+	                    },
+                      "LINE_START_TIME": {
+ 	                        "type": "number"
+	                     },
+	                     "EXPOSURE_TIME": {
+		                       "type": "number"
+	                     },
+	          "required": [ "EXPOSURE_TIME"]
+	                 }
+	              }
+      },
+       "DETECTOR_SAMPLE_SUMMING": {
+           "description": "The summing in the sample-direction",
+           "type": "number"
+       },
+       "DETECTOR_LINE_SUMMING": {
+           "description": "The summing in the line-direction",
+           "type": "number"
+       },
+       "APPLY_LIGHT_TIME_CORRECTION": {
+           "description": "A flag to indcate whether the light-time correction should be applied",
+           "type": "bool"
+       },
+       "NUMBER_OF_EPHEM": {
+           "description": "The number of emphemeris points",
+           "type": "number"
+       },
+       "NUMBER_OF_QUATERNIONS": {
+           "description": "The number of quaternions",
+           "type": "number"
+       },
+       "DT_EPHEM": {
+           "description": "The time between each ephemeris point.",
+           "type": "number"
+       },
+       "T0_EPHEM": {
+           "description": "First ephemeris time - center ephemeris time",
+           "type": "number"
+       },
+       "DT_QUAT": {
+           "description": "Time spacing of quaternions.",
+           "type": "number"
+       },
+       "T0_QUAT": {
+           "description": "First quaternion time - center quaternion time.",
+           "type": "number"
+       },
+       "REFERENCE_HEIGHT": {
+           "description": "Used for determining the intersection of the look vector with the ellipsoid",
+           "type": "array",
+           "items":  {
+              "type": "number",
+              "minItems": 2,
+              "maxItems": 2
+           }
+       },
+      "INTERPOLATION_METHOD": {
+           "description": "The type of interpolation method to use.",
+           "type": "string"
+       },
+
+
+   "required": ["FOCAL_LENGTH", "BORESIGHT","CCD_CENTER","IMAGE_LINES","IMAGE_SAMPLES" ]
```

Comment:
This is not all of the required parameters. We have to have ephemeri and the focal plane to pixel conversions at least.  There's probably more I'm missing.

Reply:
You're right this is definitely not all the required parameters! This was something that needed to be updated, thank you.